benchmarks/Throughput.h

```diff
@@ -34,15 +34,17 @@ class FixedResponder : public RSocketResponder {
 
 /// Subscriber that requests N items and cancels the subscription once all of
 /// them arrive.  Signals a latch when it terminates.
-class BoundedSubscriber : public yarpl::flowable::Subscriber<Payload> {
+class BoundedSubscriber : public yarpl::flowable::LegacySubscriber<Payload> {
  public:
   BoundedSubscriber(Latch& latch, size_t requested)
       : latch_{latch}, requested_{requested} {}
 
   void onSubscribe(
```

Comment:
Would it be enough to 
 - remove the `virtual` keyword from this function only 
 - and introduce another virtual function - which is `onSubscribe()`, which doesn't have any parameters
So it will force the child classes to call `subscription()` function to get the subscription. 
That way, we can be sure that this parent class has full autonomy over the Subscription instance and the inherited classes are forced to obey it.